src/KnownTools.ts

```diff
@@ -48,6 +48,14 @@ const knownTools = {
         fileName: "elm.exe",
         type: "gz",
       },
+      // WoW64
```

Comment:
On Windows 11, WoW64 got replaced with "Prism", but that's a generic enough name I left the comment as WoW64.